tests/vmi_configuration_test.go

```diff
@@ -475,7 +475,7 @@ var _ = Describe("[sig-compute]Configurations", func() {
 
 		Context("[rfe_id:140][crit:medium][vendor:cnv-qe@redhat.com][level:component]with no memory requested", func() {
 			It("[test_id:3113]should failed to the VMI creation", func() {
-				vmi := tests.NewRandomVMI()
+				vmi := libvmi.NewCirros()
```

Comment:
nit: you could create a local option here so it reads `libvmi.NewCirros(withoutRequests)`

Reply:
[done](https://github.com/kubevirt/kubevirt/compare/abfebeb7af24d11efe0290ac115233fb11fc91df..07e7ecbf4da4b55841e84efcfc2c1fa9cea61b1c)